### ISSUE TITLE
`CREATE_INDEX` and `CREATE_INDEX_EXCLUSIVE` docs are not helpful

### ISSUE THREAD

**Issue by sobolevn** [3YOURMIND/django-migration-linter#264]
Hi! Thanks for working on this project.
Today I updated to v5 and got some new CI failures.

Here's my migration file: https://github.com/tough-dev-school/python-testing-homework/blob/master/server/apps/identity/migrations/0001_initial.py

It generate this warning:

```
(pictures, 0001_initial)... WARNING (cached)
        CREATE INDEX prolongs transaction, delaying lock release
```

It is not clear from the docs on how to fix this warning. This forced me to switch it off for now.

**Comment by David-Wobrock** [comment]
Hey @sobolevn 

Fair point, thanks for raising it 👍 
I'll tackle that :)

**Comment by cconn** [comment]
Hi, a problem I've observed is that we often get false positives when creating a table and index in the same transaction. It looks like this is a regression of #178. I think @sobolevn's warning might also have been a false positive?

I've opened a PR to address this: https://github.com/3YOURMIND/django-migration-linter/pull/295
